pandas/tests/arrays/string_/test_string.py

```diff
@@ -431,23 +431,25 @@ def test_arrow_array(dtype):
 
 
 @td.skip_if_no("pyarrow")
-def test_arrow_roundtrip(dtype):
+def test_arrow_roundtrip(dtype, string_storage2):
```

Comment:
the skips should be encompassed in the fixtures themselves no? (can change this later)

Reply:
this test is skipped if pyarrow not installed since the test needs pyarrow to create `pa.table(df)`, and skips the ArrowStringArray tests from fixture if the installed version of pyarrow is < 1.0.0, so would only test the python StringDtype with the python global setting